lms/static/sass/course/layout/_reset_deadlines.scss

```diff
@@ -11,12 +11,13 @@ div.reset-deadlines-banner {
 
     &.reset-deadlines-text {
       color: theme-color("inverse");
-      padding-top: 2px;
+      padding-top: 10px;
       margin-right: 10px;
     }
 
     &.reset-deadlines-button {
-      border-radius: 5px;
+      color: #0075b4;
```

Comment:
Is there any other way to specify this color? Or at the very least, adding in a comment for what this is.